tests/utils.go

```diff
@@ -56,19 +30,12 @@ func CDIFailHandler(message string, callerSkip ...int) {
 	Fail(message, callerSkip...)
 }
 
-func RunKubectlCommand(args ...string) (string, error) {
-	kubeconfig := flag.Lookup("kubeconfig").Value
-	if kubeconfig == nil || kubeconfig.String() == "" {
-		return "", fmt.Errorf("can not find kubeconfig")
-	}
-
-	master := flag.Lookup("master").Value
-	if master != nil && master.String() != "" {
-		args = append(args, "--server", master.String())
-	}
+func RunKubectlCommand(f *framework.Framework, args ...string) (string, error) {
```

Comment:
So if I understand correctly, now the framework can never call RunKubectlCommand because that would generate a circular dependency. Also if I understand correctly, as long as flag.Parse() is called I can then use flag.lookup for basically anywhere. So why not simply call flag.Parse in NewFramework, and leave the flag.lookup in here? That way we can call RunKubectlCommand from Framework if we ever need to.